test/parallel/test-whatwg-encoding-fatal-streaming.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+// From: https://github.com/w3c/web-platform-tests/blob/d74324b53ca6ebf9ffd51333c216cadfa6ea2303/encoding/textdecoder-fatal-streaming.html
```

Comment:
We can use the first 7 characters of the SHA if we don't want a really long URL here. That works as well.

Reply:
I usually prefer 10 characters, as that's what modern Git outputs IIRC.

Reply:
Oh, right, forgot about those :-)